src/panels/config/automation/trigger/types/ha-automation-trigger-webhook.ts

```diff
@@ -1,24 +1,83 @@
 import "@polymer/paper-input/paper-input";
-import { html, LitElement } from "lit";
-import { customElement, property } from "lit/decorators";
-import { WebhookTrigger } from "../../../../../data/automation";
+import "../../../../../components/ha-icon-button";
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import { mdiContentCopy } from "@mdi/js";
+import { css, html, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import type { PaperInputElement } from "@polymer/paper-input/paper-input";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import { slugify } from "../../../../../common/string/slugify";
+import { copyToClipboard } from "../../../../../common/util/copy-clipboard";
+import { showToast } from "../../../../../util/toast";
+import {
+  WebhookTrigger,
+  AutomationConfig,
+} from "../../../../../data/automation";
 import { HomeAssistant } from "../../../../../types";
 import { handleChangeEvent } from "../ha-automation-trigger-row";
 
+const DEFAULT_WEBHOOK_ID = "default webhook id";
+
 @customElement("ha-automation-trigger-webhook")
 export class HaWebhookTrigger extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
   @property() public trigger!: WebhookTrigger;
 
+  @state() private _config?: AutomationConfig;
+
+  private _unsub?: UnsubscribeFunc;
+
   public static get defaultConfig() {
     return {
-      webhook_id: "",
+      webhook_id: DEFAULT_WEBHOOK_ID,
     };
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+    const details = { callback: (config) => this._automationUpdated(config) };
```

Comment:
You can inline this function.
```suggestion
    const details = { callback: (config) => { this._config = config; } };
```